en/lessons/basics/operators.md

```diff
@@ -0,0 +1,164 @@
+---
+version: 1.0.0
+title: Operators
+---
+
+Haskell provides constructs for using infix binary functions, called operators. In
+this lesson you will learn how to define operators and control the parsing of
+operators, and how infix and prefix syntax can be interchanged.
+
+{% include toc.html %}
+
+
+## Using Operators
+
+Operators in Haskell are functions that can be placed between two arguments: they can
+return a value of any type and their names are either a string of symbols or ordinary
+function names surrounded in backticks. Using infix notation makes it easier to write
+expressions containing many common functions:
+
+```console?lang=haskell&prompt=ghci>,ghci|
+ghci> abs . subtract 930233356 $ 2 + 2
+930233352
+ghci> (`mod` 1119) . negate $ 4
+1115
+ghci> - (7 + 57843)
+-57850
+```
+__Note__: negate (`-`) is the only unary operator in Haskell.
+
+You may have seen operators such as `+`, `*`, and `.` before. The `$` operator is
+useful for removing brackets in function application: `abs . subtract 1 $ 2 + 2` is
+equivalent to `(abs . subtract 1) (2 + 2)`.
+
+### Prefix and Infix Notation
+
+To use an operator symbol in prefix notation, as if it were a regular function,
+surround it in brackets. Furthermore, you can create partially applied operators by
+putting an expression on only one side of the operator:
+
+```console?lang=haskell&prompt=ghci>,ghci|
+ghci> (+) 2 3
+5
+ghci> (5 +) . (* 2) $ 8
+21
+```
+
+Conversely, to make a regular function identifier an operator, surround it in backticks:
+
+```console?lang=haskell&prompt=ghci>,ghci|
+ghci> 2010 `mod` 1998
+12
+ghci> (`div` 3) . (* 1471684) $ 5
```

Comment:
The usage of the concept of interchange may be confusing for the reader, given that this process is applied twice here: the example makes an ordinary function identifier into an operator, and then back into a prefix function that is partially applied.